security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -1592,6 +1592,11 @@
                     <mask>/^.{1,1024}$/u</mask>
                     <ValidationMessage>Should be a string between 1 and 1024 characters.</ValidationMessage>
                 </acme_vault_prefix>
+                <acme_vault_token type="TextField">
+                                    <Required>Y</Required>
+                                    <mask>/^.{1,1024}$/u</mask>
+                                    <ValidationMessage>Should be a string between 1 and 1024 characters.</ValidationMessage>
```

Comment:
The indention of these lines is off. You probably used tabs but should use spaces instead.